optimum/onnxruntime/trainer.py

```diff
@@ -443,7 +544,7 @@ def _inner_training_loop(
                     RuntimeWarning,
                 )
 
-            self.model = model
+            self.model = self._training_model
```

Comment:
once ORTModule wrap training model complete, where is the ortmodule instance stored? How could we get it next time for training?

Reply:
OrtModule instance won't be stored, we will wrap the training_model again on every _inner_training_loop called